test/unit/adapter/base_tests.rb

```diff
@@ -115,41 +115,19 @@ class ConnectDbTests < UnitTests
     end
   end
 
-  class MigrateDbTests < UnitTests
-    desc "`migrate_db`"
+  class MigrateTests < UnitTests
     setup do
-      @orig_migrate_version_env_var = ENV["MIGRATE_VERSION"]
-      @orig_migrate_query_env_var   = ENV["MIGRATE_QUIET"]
-
-      ENV["MIGRATE_VERSION"] = Factory.integer.to_s if Factory.boolean
-      ENV["MIGRATE_QUIET"]   = Factory.boolean.to_s if Factory.boolean
-
-      @migrator_called_with = []
-      Assert.stub(ActiveRecord::Migrator, :migrate) do |*args|
-        @migrator_called_with = args
+      skip
```

Comment:
@kellyredding - Are you planning on addressing this in a future commit or did you want to handle it here?

Reply:
No, forgot about this so thanks - I need to deal with it here - stay tuned.

Reply:
Ok, I've added proper tests for the new migrate commands - please look again thanks!